backend/bin/server/cmd/profile.go

```diff
@@ -24,6 +24,7 @@ func getBaseProfile(dataDir string) config.Profile {
 		ExternalURL:          flags.externalURL,
 		GrpcPort:             flags.port + 1, // Using flags.port + 1 as our gRPC server port.
 		DatastorePort:        flags.port + 2, // Using flags.port + 2 as our datastore port.
+		LSPProt:              flags.port + 4, // Using flags.port + 4 as our LSP server port.
```

Comment:
Can we reuse the same port?

Reply:
Guess that's possible since `AdminExecute` uses WebSocket and shares the same port with grpc requests.